src/style/style.less

```diff
@@ -53,6 +66,47 @@ body {
   flex-direction: column;
 }
 
+.btn {
+  display: inline-block;
+  padding: 6px 12px;
+  margin-bottom: 0;
+  font-size: 14px;
+  font-weight: normal;
+  line-height: 1.428571429;
+  text-align: center;
+  vertical-align: middle;
+  cursor: pointer;
+  border: 1px solid transparent;
+  border-radius: 4px;
+  white-space: nowrap;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  -o-user-select: none;
+  user-select: none;
+
+  @media screen and (max-width: @mobileWidth) {
+    padding: 5px 10px;
+    font-size: 12px;
+    line-height: 1.5;
+    border-radius: 3px;
+    margin-top: 8px;
+  }
+}
+
+.btn-primary {
+  color: #ffffff;
+  background-color: @mainColor;
+  border-color: @mainColor;
+}
+
+.container {
```

Comment:
All btn, btn-primary and container styles are already being defined by Bootstrap. Do we need them here?